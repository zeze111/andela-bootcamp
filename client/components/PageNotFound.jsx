import React from 'react';

/**
 *
 *
 * @returns {function} App
 * @extends {React.Component}
 */
const PageNotFound = () => (
  <div className="row">
    <div className="col s12 center-align none">
      <h3 className="nf-text home-width"> Oops! Page Not Found </h3>
    </div>
    <div className="col s12 center-align nf-page">
      <img
        alt="Not Found"
        src="/images/404.jpg"
        className="responsive-img"
      />
    </div>
    <div className="col s12 center-align none">
      <h5 className="nf-text2"> When the page you're looking for
      gives you lemons,
      </h5>
      <h5 className="nf-text3"> make
        <span className="nf-span"> Lemonade </span>
      </h5>
    </div>
  </div>
);

export default PageNotFound;
