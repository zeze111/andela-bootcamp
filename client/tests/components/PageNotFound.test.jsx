import React from 'react';

import PageNotFound from '../../components/PageNotFound';

describe('Test for 404 Page', () => {
  it('', () => {
    const wrapper = shallow(<PageNotFound /> );
    expect(wrapper).toMatchSnapshot()
  });
});
