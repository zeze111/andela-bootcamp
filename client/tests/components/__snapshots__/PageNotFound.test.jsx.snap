// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for 404 Page  1`] = `
<div
  className="row"
>
  <div
    className="col s12 center-align none"
  >
    <h3
      className="nf-text home-width"
    >
       Oops! Page Not Found 
    </h3>
  </div>
  <div
    className="col s12 center-align nf-page"
  >
    <img
      alt="Not Found"
      className="responsive-img"
      src="/images/404.jpg"
    />
  </div>
  <div
    className="col s12 center-align none"
  >
    <h5
      className="nf-text2"
    >
       When the page you're looking for gives you lemons,
    </h5>
    <h5
      className="nf-text3"
    >
       make
      <span
        className="nf-span"
      >
         Lemonade 
      </span>
    </h5>
  </div>
</div>
`;
